README.md

```diff
@@ -63,32 +63,27 @@ Example URL with query parameters:
 
 Valid properties for model specification referenced by `model_url`:
 
-* model_id - (string) The id of the model
-* model_caption - (string) The caption of the model
+* RID - (string) The id of the model
 * bg_color_r, bg_color_g, bg_color_b (int [0-255]) - RGB values for the background
 * bounding_box_color_r, bounding_box_color_g, bounding_box_color_b (int [0-255]) -
  Color of the Bounding Box
-* model_measurement - (string [inches, cm]) units of distance measurement
-* model_unitconversion - (float [1.0]) Multiplied by world-space distance to
+* measurement - (string [inches, cm]) units of distance measurement
```

Comment:
let's make one change here. the `measurement` should be renamed `units` and it should default to something (which I need to confirm). probably to `mm` if this property is not given or is null.

Reply:
The default is currently 'units', which was intentionally vague to show the type was unknown so it wasn't accidentally mistaken for a different kind. I can switch that to 'mm' though.

Reply:
yes, please make it 'mm' and add that to the documentation here.